server/message_hooks.go

```diff
@@ -586,7 +587,9 @@ func (p *Plugin) Update(teamID, channelID string, user *model.User, newPost, old
 
 	if err = client.UpdateMessage(teamID, channelID, parentID, postInfo.MSTeamsID, content, mentions); err != nil {
 		p.API.LogWarn("Error updating the post", "error", err)
-		return err
+		if !strings.Contains(err.Error(), "code: PaymentRequired") {
```

Comment:
This looks like "masking" the error, instead of fixing the error. When this happens and is safe to ignore it? Should we at least log why this is happening?

Reply:
There's a separate issue created #152 for exploring this error. Here, we are ignoring it because the post is still getting updated in spite of the error.